python/example/network_ring.py

```diff
@@ -87,15 +87,15 @@ def cell_kind(self, gid):
     # (8) Make a ring network. For each gid, provide a list of incoming connections.
     def connections_on(self, gid):
         src = (gid-1)%self.ncells
-        w = 0.01
+        w = 0.01 # μS on expsyn
```

Comment:
Just to make it clear, I'd propose the comment `# 0.01 μS on expsyn` here and below.